serverless/admin_guide/serverless-ha.adoc

```diff
@@ -9,10 +9,19 @@ toc::[]
 High availability (HA) is a standard feature of Kubernetes APIs that helps to ensure that APIs stay operational if a disruption occurs.
 In an HA deployment, if an active controller crashes or is deleted, another controller is available to take over processing of the APIs that were being serviced by the controller that is now unavailable.
 
-HA in {ServerlessProductName} is available through leader election, which is enabled by default after the Knative Serving control plane is installed.
+HA in {ServerlessProductName} is available through leader election, which is enabled by default after the Knative Serving or Eventing control plane is installed.
 
 When using a leader election HA pattern, instances of controllers are already scheduled and running inside the cluster before they are required.
 These controller instances compete to use a shared resource, known as the leader election lock.
 The instance of the controller that has access to the leader election lock resource at any given time is referred to as the leader.
 
-include::modules/serverless-config-replicas.adoc[leveloffset=+1]
+== Configuring high availability replicas on {ServerlessProductName}
+
+High availability (HA) functionality is available by default on {ServerlessProductName} for the `autoscaler-hpa`, `controller`, `activator`, `kourier-control`, and `kourier-gateway` controllers. These components are configured with two replicas by default.
```

Comment:
This list may be inaccurate. As we now also have HA for eventing and knative kafka, they have their own controllers and webhooks, which are also HA now. Also, "activator" and "kourier-gateway" are not technically "controllers".

Reply:
@maschmid Can you please provide the full updated list?

Reply:
For KnativeServing:

activator, autoscaler, autoscaler-hpa, controller, webhook, kourier-control, kourier-gateway

For KnativeEventing:

eventing-controller, eventing-webhook, imc-controller, imc-dispatcher, mt-broker-controller, sugar-controller

(notably, mt-broker-filter and mt-broker-ingress deployments are not scaled according to KnativeEventing `replicas`
 setting, and the deployments need to be scaled manually, if needed. In upstream they're scaled automatically by an HorizontalPodAutoscaler)
 
For KnativeKafka:

kafka-ch-controller, kafka-controller-manager, kafka-webhook

Reply:
Thanks for the list, implemented.